engine/verification/test/assignerfetcher_test.go

```diff
@@ -44,7 +29,7 @@ import (
 func TestVerificationHappyPath(t *testing.T) {
 	testcases := []struct {
 		blockCount      int
-		opts            []vertestutils.CompleteExecutionReceiptBuilderOpt
+		ops             []vertestutils.CompleteExecutionReceiptBuilderOpt
```

Comment:
```suggestion
		opts             []vertestutils.CompleteExecutionReceiptBuilderOpt
```
Maybe overwriting a commit from previous PR? Pretty sure I've made this suggestion previously 😅